pallets/payment-streams/src/lib.rs

```diff
@@ -229,20 +227,15 @@ pub mod pallet {
         #[pallet::weight(Weight::from_parts(10_000, 0) + T::DbWeight::get().writes(1))]
         pub fn create_payment_stream(
```

Comment:
Why don't we remove this extrinsic and utilize unit tests to test `do_create_payment_stream` and eventually integration tests? I found it confusing that this was here and it confirmed my suspicion after seeing the docs for this stating it was for testing purposes only.

Reply:
This is not only for rust tests but also for manual tests, but I do agree that considering that these extrinsics should be removed before deploying in production it does not make a lot of sense to use them in the unit tests, so I will change them to use the `PaymentStreamsInterface` functions instead